templates/pages/admin/user.substitute.html.twig

```diff
@@ -0,0 +1,130 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2022 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+{% import 'components/form/fields_macros.html.twig' as fields %}
+
+{% set id      = item.fields['id'] ?? -1 %}
+
+<div class="asset">
+
+{% if canedit %}
+   <form name="asset_form" method="post" action="{{ item.getFormURL() }}" enctype="multipart/form-data" data-submit-once>
+{% endif %}
+
+   <div id="mainformtable">
+
+      {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::PRE_ITEM_FORM'), {'item': item, 'options': params}) }}
+
+      <div class="card-body d-flex flex-wrap">
+         <div class="col-12 col-xxl-{{ item_has_pictures ? '9' : '12' }} flex-column">
+            <div class="d-flex flex-row flex-wrap flex-xl-nowrap">
+               <div class="row flex-row align-items-start flex-grow-1">
+                  <div class="row flex-row">
+                     <input type="hidden" name="users_id" value={{ user.fields['id'] }} />
+
+                     {{ fields.datetimeField(
+                        'substitution_start_date',
+                        user.fields['substitution_start_date'],
+                        __('Start date ')
+                     ) }}
+
+                     {{ fields.datetimeField(
+                        'substitution_end_date',
+                        user.fields['substitution_end_date'],
+                        __('End date ')
+                     ) }}
+
+                     {{ fields.dropdownField(
```

Comment:
I think substitutes field is the more important one of the form and should be place at the top.

Reply:
I placed the dates on top because I think that the substitutes will be often set the first time the user will activate the feature. The next times, the user may just update the dates, without changing the substitutes (assument there is not too much turnover in the company). In such use case, dates becomes more important than substitutes.